### DISCUSSION TITLE
How to get feature property in cluster based on another property value?

### DISCUSSION THREAD

**Discussion by makso94** [maplibre/maplibre-gl-js#4699] [Q&A]
I have features that have an svg picture as a property, as well as alert status. So, when these features are clustered, I want to get the cluster image from the feature with the highest alert status. How do I express this in ClusterProperties?

**Comment by HarelM** [comment]
You'll need to dig into the cluster capabilities, which is mostly decided by supercluster and geojson-vt.
I haven't tried complicated stuff when it comes to clustering, so you'll need to dig into it yourself I guess...
